### PR TITLE
Development openrpcdoc fix

### PR DESCRIPTION
**Pull request by timurgordon**
Solves #31 and #40

### CODE REVIEW

**Comment by DylanVerstraete** [comment]
Can we not include the built playground files?

File: doc.sh [hunk 1]
```diff
@@ -7,6 +7,25 @@ pushd manual
 bash run.sh
 popd
 
+# compile openrpc cli binary
+v ~/.vmodules/freeflowuniverse/crystallib/openrpc/cli
+OPENRPC_CLI=~/.vmodules/freeflowuniverse/crystallib/openrpc/cli/cli
+
+# generate doc for entire api
+$OPENRPC_CLI docgen -t "Web3Proxy JSON-RPC API" -p -o server lib
```
Comment (brandonpille) [inline doc.sh:15]: We still need that if we do them individually?

**Review by brandonpille — APPROVED** [review]
Looks good. Do we still need the openrpc.json files in the repository? As they are generated by CI I would remove them from the repository.

File: doc.sh (hunk 1 above)
Comment (timurgordon) [reply doc.sh:15]: not really, but why not. I link to it's playground in the ui, maybe it interests some people to see the full scope of the api. I'm ok with removing it

Comment (brandonpille) [reply doc.sh:15]: Ok. Let's keep it until we have a reason not too

**Pull request merged**